signing_service/signing_service/tests/tests.py

```diff
@@ -7,29 +7,38 @@
 import tempfile
 
 from golem_messages.cryptography import ECCx
+from golem_messages.cryptography import ecdsa_sign
 from golem_messages.message import Ping
 import assertpy
 import mock
 import pytest
 
+from middleman_protocol.concent_golem_messages.message import SignedTransaction
+from middleman_protocol.concent_golem_messages.message import TransactionRejected
 from middleman_protocol.constants import ErrorCode
+from middleman_protocol.constants import FRAME_PAYLOAD_STARTING_BYTE
+from middleman_protocol.constants import FRAME_PAYLOAD_TYPE_LENGTH
+from middleman_protocol.constants import FRAME_REQUEST_ID_BYTES_LENGTH
+from middleman_protocol.constants import FRAME_SIGNATURE_BYTES_LENGTH
 from middleman_protocol.message import AbstractFrame
 from middleman_protocol.message import GolemMessageFrame
-from middleman_protocol.stream import append_frame_separator
-from middleman_protocol.stream import escape_encode_raw_message
 from middleman_protocol.stream import unescape_stream
 
 from signing_service.constants import REQUEST_ID_FOR_RESPONSE_FOR_INVALID_FRAME
 from signing_service.constants import SIGNING_SERVICE_DEFAULT_PORT
 from signing_service.constants import SIGNING_SERVICE_DEFAULT_INITIAL_RECONNECT_DELAY
-from signing_service.constants import SIGNING_SERVICE_RECOVERABLE_ERRORS
 from signing_service.constants import SIGNING_SERVICE_MAXIMUM_RECONNECT_TIME
+from signing_service.constants import SIGNING_SERVICE_RECOVERABLE_ERRORS
 from signing_service.exceptions import SigningServiceValidationError
 from signing_service.signing_service import _parse_arguments
 from signing_service.signing_service import SigningService
-from signing_service.utils import is_valid_public_key
+from signing_service.utils import is_public_key_valid
+
+from .utils import SigningServiceIntegrationTestCase
 
 
+TEST_ETHEREUM_PRIVATE_KEY = '3a1076bf45ab87712ad64ccb3b10217737f7faacbf2872e88fdd9a537d8fe266'
```

Comment:
Please use `secret` package to generate random keys. To this please use:
```bash
>>> from secrets import token_hex
>>> token_hex()
'8b98656db2e98a5c94ef80e16d082d5d1ff814f77cf6de00115d2a6b5f38c748'
>>> len(token_hex())
64
```

Reply:
It can't be generated, because it will result in different signatures each time.